exporter/loggingexporter/internal/otlptext/databuffer.go

```diff
@@ -273,17 +273,17 @@ func (b *dataBuffer) logLinks(description string, sl ptrace.SpanLinkSlice) {
 func attributeValueToString(v pcommon.Value) string {
 	switch v.Type() {
 	case pcommon.ValueTypeString:
-		return v.StringVal()
+		return v.Str()
 	case pcommon.ValueTypeBool:
-		return strconv.FormatBool(v.BoolVal())
+		return strconv.FormatBool(v.Bool())
 	case pcommon.ValueTypeDouble:
-		return strconv.FormatFloat(v.DoubleVal(), 'f', -1, 64)
+		return strconv.FormatFloat(v.Double(), 'f', -1, 64)
 	case pcommon.ValueTypeInt:
-		return strconv.FormatInt(v.IntVal(), 10)
+		return strconv.FormatInt(v.Int(), 10)
 	case pcommon.ValueTypeSlice:
-		return sliceToString(v.SliceVal())
+		return sliceToString(v.Slice())
 	case pcommon.ValueTypeMap:
-		return mapToString(v.MapVal())
+		return mapToString(v.Map())
 	default:
```

Comment:
This lacks the ValueTypeBytes? I will take care of this in a separate PR.

Reply:
Yes, nice catch.